webui/src/styles/globals.css

```diff
@@ -1,928 +1,228 @@
+  --bs-nav-link-hover-color: #2563eb;
+  --danger: #ef4444;
+  --danger-light: #f87171;
+  --danger-dark: #dc2626;
+  --warning: #f59e0b;
+  --warning-light: #fbbf24;
+  --warning-dark: #d97706;
+  --info: #06b6d4;
+  --info-light: #22d3ee;
+  --info-dark: #0891b2;
+  --background: #ffffff;
+  --background-alt: #f9fafb;
+  --surface: #ffffff;
+  --text: #1f2937;
+  --text-light: #6b7280;
+  --text-dark: #111827;
+  --border: #e5e7eb;
+  --border-light: #f3f4f6;
+  --border-dark: #d1d5db;
+
+  /* Semantic Colors */
+  --color-fg-added: var(--success);
+  --color-fg-removed: var(--danger);
+  --color-fg-changed: var(--warning);
+  --color-fg-conflict: var(--secondary);
+  --color-uri-octicon: var(--primary);
+  --color-uri-link: var(--primary-dark);
+  --color-bg-added: #ecfdf5;
+  --color-bg-removed: #fef2f2;
+  --color-bg-changed: #fffbeb;
+  --color-bg-unknown: #eff6ff;
   --color-bg-conflict: #fff3cd;
-  --color-policy-text: #d63384;
-/* The following is all the colors that are used by the css (kept here for refernce, when adding new colors): */
-/*  --bs-success: var(--bs-success);*/
-/*  --bs-danger: var(--bs-danger);*/
-/*  --bs-warning: var(--bs-warning);*/
-/*  --bs-body-bg: var(--bs-body-bg);*/
-/*  --bs-tertiary-bg: var(--bs-tertiary-bg);*/
-/*  --bs-light-border-subtle: var(--bs-light-border-subtle);*/
-/*  --bs-border-color: var(--bs-border-color);*/
-/*  --bs-dark-border-subtle: var(--bs-dark-border-subtle);*/
-/*  --bs-secondary: var(--bs-secondary);*/
-/*  --bs-light-text-emphasis: var(--bs-light-text-emphasis);*/
-/*  --bs-dark-text-emphasis: var(--bs-dark-text-emphasis);*/
-/*  --bs-body-color: var(--bs-body-color);*/
-/*  --bs-emphasis-color: var(--bs-emphasis-color);*/
-/*  --bs-blue: var(--bs-blue);*/
-/*  --bs-cyan: var(--bs-cyan);*/
-/*  --bs-pink: var(--bs-pink);*/
-/*  --bs-teal: var(--bs-teal);*/
+  --color-policy-text: #be185d;
+
+  /* Shadows */
+  --shadow-sm: 0 1px 2px 0 rgba(0, 0, 0, 0.05);
+  --shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
+  --shadow-md: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
+  --shadow-lg: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05);
+  
+  /* Spacing */
+  --spacing-xs: 0.25rem;
+  --spacing-sm: 0.5rem;
+  --spacing-md: 1rem;
+  --spacing-lg: 1.5rem;
+  --spacing-xl: 2rem;
+  --spacing-2xl: 3rem;
+  
+  /* Border Radius */
+  --radius-sm: 0.25rem;
+  --radius: 0.375rem;
+  --radius-md: 0.5rem;
+  --radius-lg: 0.75rem;
+  --radius-xl: 1rem;
+  --radius-full: 9999px;
+  
+  /* Typography */
+  --font-sans: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+  --font-mono: ui-monospace, SFMono-Regular, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+  
+  /* Transitions */
+  --transition-fast: 150ms cubic-bezier(0.4, 0, 0.2, 1);
+  --transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
+  --transition-slow: 300ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
This part is probably the best part of this PR - 
And I believe we would benefit only from keeping this (after well testing it, of course).

It provides a general style that we're currently missing.